src/activities/assignments/AssignmentEntity.js

```diff
@@ -610,6 +614,22 @@ export class AssignmentEntity extends Entity {
 			fields.push({ name: 'completionType', value: assignment.completionType });
 		}
 
+		if (typeof assignment.isIndividualAssignmentType !== 'undefined') {
+			if (assignment.isIndividualAssignmentType !== this.isIndividualAssignmentType()) {
+				fields.push({ name: 'isIndividualAssignmentType', value: assignment.isIndividualAssignmentType });
```

Comment:
This field is not present on the [controller definition](https://search.d2l.dev/xref/lms/le/dropbox/D2L.LE.Dropbox.Extensibility/HM/Folders/Controllers/FoldersController.cs?r=65c56194#215). What are you trying to do here?